ci-operator/jobs/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.9-presubmits.yaml

```diff
@@ -16,6 +16,7 @@ presubmits:
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-windows-machine-config-bootstrapper-community-4.9-okd-build
     rerun_command: /test okd-build
+    skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
```

Comment:
This needs to be done for the community-4.9 WMCO branch as well

Reply:
Added